NAMESPACE

```diff
@@ -44,6 +44,7 @@ export(getUniqueTaxa)
 export(inner_join)
 export(left_join)
 export(loadFromBiom)
+export(loadFromHumann)
```

Comment:
I think this name is ok although also forms "HUMAnN" and "humann" are used in the original sources (see e,g, https://github.com/biobakery/humann)

Reply:
Yep, our function naming follows this scheme so I agree that this is ok. (I think loadFromHUMAnN might be too hard to remember which letters are in uppercase and which not) --> we can change this also in the future if needed